pkg/networkserver/grpc_gsns.go

```diff
@@ -359,7 +358,15 @@ func (ns *NetworkServer) handleUplink(ctx context.Context, up *ttnpb.UplinkMessa
 	pld := up.Payload.GetMACPayload()
 
 	logger := log.FromContext(ctx).WithFields(log.Fields(
```

Comment:
Perhaps we should create a shared func for getting the log fields from a LoRaWAN payload, so that we have consistent log fields across GS/NS/AS.